src/view/WorkspaceManager.js

```diff
@@ -191,6 +201,7 @@ define(function (require, exports, module) {
      */
     Panel.prototype.hide = function () {
         Resizer.hide(this.$panel[0]);
+        exports.trigger(EVENT_WORKSPACE_PANEL_HIDDEN, this.panelID);
```

Comment:
In the case, say if we raise a panel visibilitychanged event, then we would need to write an intermediate function to route that event to `PanelShown` and `panelHidden` functions. That can be avoided if we can directly tag a specific `trigger` to an `action`. Maybe just a choice of writing style :)